setup.py

```diff
@@ -23,6 +23,7 @@
     "flax>=0.3.0, <0.4",
     "orjson~=3.4",
     "optax>=0.0.2, <=0.0.7",
+    "compose>=1.1.1",
```

Comment:
what is compose?

Reply:
is `compose(a,b)` roughly equivalent to `lambda *args, **kwargs: a(b(*args, **kwargs))` ?

Anyhow, if I am not mistaken it is  not on `conda-forge` which is a bit unfortunate

Reply:
I mean I love this kind of thing. I also think it makes the code cleaner.
I am not opposed to it, but another dependency (which we will have to also bring to conda-forge) I don't know. 
Before merging this I'd like the recipe to be available on conda-forge at least.

Is there any alternative a bit more high profile? Or should we rather we vendor it in `netket.utils` and its tests than add it as a dependency...

Reply:
> is `compose(a,b)` roughly equivalent to `lambda *args, **kwargs: a(b(*args, **kwargs))` ?

It's not used for anything else at least... we could just add this definition somewhere and not generate more dependencies